src/main/java/seedu/address/commons/util/JsonUtil.java

```diff
@@ -90,6 +90,22 @@
         serializeObjectToJsonFile(filePath, jsonFile);
     }
 
+    /**
+     * Saves the Json object to the specified file.
+     * Overwrites existing file if it exists, creates a new file if it doesn't.
+     * @param jsonFile cannot be null
+     * @param filePath cannot be null
+     * @throws IOException if there was an error during writing to the file
+     */
+    public static <T> void saveJsonFile(T jsonFile, T jsonFile2, Path filePath) throws IOException {
+        requireNonNull(filePath);
+        requireNonNull(jsonFile);
+        requireNonNull(jsonFile2);
+
+        serializeObjectToJsonFile(filePath, jsonFile);
+        serializeObjectToJsonFile(filePath, jsonFile2);
```

Comment:
this method is not needed, will have to delete

Reply:
this method is not called at all, its overloaded one is called (and used every time the app is closed)